test/e2e/syncer/syncer_test.go

```diff
@@ -238,8 +213,18 @@ func TestSyncerLifecycle(t *testing.T) {
 			return false
 		}
 		return true
-	}, wait.ForeverTestTimeout, time.Millisecond*100, "downstream deployment %s/%s was not synced", downstreamNamespaceName, upstreamDeployment.Name)
+	}, 60*time.Second, time.Millisecond*100, "downstream deployment %s/%s was not synced", downstreamNamespaceName, upstreamDeployment.Name)
```

Comment:
Why'd you change this to 60 seconds?

Reply:
it's a wip commit, im trying to find why some tests are flaky

Reply:
The flakes are most likely unrelated. I think they came from #1082.